integr8ly/tower_dummy_credentials
# Updated process to encrypt the credential_config.yml

## Additional Information

https://issues.jboss.org/browse/INTLY-4099

This PR encrypts the `CREDENTIAL_CONFIG.yml` file. This allows the file to be checked into a repository, thus a single copy of the `CREDENTIAL_CONFIG.yml` can be shared and used between a team, as opposed to each team member having their own copy of the file.

## Verification Steps
1.  Using this feature branch, execute the `bootstrap.yml` playbook as per the [README.md](https://github.com/obrienrobert/tower_dummy_credentials#setup). Once complete, ensure that the `CREDENTIAL_CONFIG.yml` is encrypted.

2. Decrypt the file as per the [instructions](https://github.com/obrienrobert/tower_dummy_credentials#files), using the same password that was used to encrypt.

3. Make some changes to the variables in the `CREDENTIAL_CONFIG.yml` file and re-run the `bootstrap.yml` playbook.

4. Repeat step 2, ensuring that the new variable values are in the `CREDENTIAL_CONFIG.yml` file when decrypted.
